src/unity/schemas/OrganizationSummary.yml

```diff
@@ -21,4 +21,7 @@ properties:
   regionName:
     type: string
     description: the friendly name of the region where the organization is hosted
+  state:
+    $ref: './OrganizationState.yml'
+    description: the current state of the organization
```

Comment:
```suggestion
    description: Current state of the organization
```

Reply:
Thanks for the feedback! I got those changes added!